services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/StaticTableMetadata.java

```diff
@@ -166,37 +171,31 @@ public int hashCode() {
         return result;
     }
 
-    @ThreadSafe
+    /**
+     * Builder for {@link StaticTableMetadata}
+     */
     public static class Builder {
-        private final Map<String, Object> customMetadata = new ConcurrentHashMap<>();
-        private final Map<String, Index> indexByNameMap = new ConcurrentHashMap<>();
-        private final Map<String, AttributeValueType> keyAttributes = new ConcurrentHashMap<>();
+        private final Map<String, Object> customMetadata = new HashMap<>();
```

Comment:
Why did you change to HashMap?

Reply:
I realized there was no value in maintaining this class as threadsafe as it was an immutable class and we can just follow our standard patterns.

Reply:
Well to be more specific we don't require any of our builders to be threadsafe. It's generally well understood they are not.